bootstrap/kubeadm/main.go

```diff
@@ -78,14 +78,14 @@ func InitFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&enableLeaderElection, "enable-leader-election", false,
 		"Enable leader election for controller manager. Enabling this will ensure there is only one active controller manager.")
 
-	fs.DurationVar(&leaseDuration, "lease-duration", 15*time.Second,
-		"Interval at which non-leader candidates will wait to force acquire leadership (e.g. 20s)")
+	fs.DurationVar(&leaderElectionLeaseDuration, "lease-duration", 15*time.Second,
```

Comment:
```suggestion
	fs.DurationVar(&leaderElectionLeaseDuration, "leader-election-lease-duration", 15*time.Second,
```

& others as well

Reply:
dang didn't even get to make my second commit!